tests/test_state_estimators.py

```diff
@@ -237,7 +246,6 @@ def test_PF_incorrect_input(self):
         self.__incorrect_input_tests(ParticleFilter)
 
     def test_KF(self):
-        from prog_algs.state_estimators import KalmanFilter
         from prog_models import LinearModel
```

Comment:
```suggestion
```

I just realized this is double imported. The only thing that should be imported in the functions is the class under test (KF here). This was my mistake from earlier. Can be fixed in test_UKF as well